.github/workflows/sync-to-oss.yml

```
@@ -0,0 +1,314 @@
+          MIT © Bonginkan Inc.
+          EOF
+          fi
+          
+          # Update package.json for OSS distribution
+          echo "📦 Updating package.json for OSS..."
+          if [ -f "/tmp/maria-oss/package.json" ]; then
+            # Remove private field and internal dependencies
+            node -e "
+              const fs = require('fs');
+              const pkg = JSON.parse(fs.readFileSync('/tmp/maria-oss/package.json', 'utf8'));
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)